TutorialApplication/OrganizationInfoViewController.swift

```diff
@@ -8,10 +8,49 @@
 
 import UIKit
 
-class OrganizationInfoViewController: UIViewController
+@objc class OrganizationInfoViewController: UIViewController
 {
     var organization: FFOrganization!
+    
+    let kEmployeesOrderHasChanged:String = "employeesOrderHasChanged"
```

Comment:
let kEmployeesOrderHasChanged:String = "employeesOrderHasChanged"
change to
static let kEmployeesOrderHasChanged= "EmployeesOrderHasChanged"